rasterio/rio/bounds.py

```diff
@@ -104,13 +113,12 @@ def __call__(self):
                 self._xs.extend(bbox[::2])
                 self._ys.extend(bbox[1::2])
 
-    try:
-        with ctx.obj['env'] as env:
-            write_features(
-                stdout, Collection(env), sequence=sequence,
-                geojson_type=geojson_type, use_rs=use_rs,
-                **dump_kwds)
-
-    except Exception:
```

Comment:
@snowman2 this is the the cause of the error message we see when the interpreter shuts down. When we catch BrokenPipeError here, Python is out of whack.

It's a reminder to not catch exceptions that aren't raised on purpose.

Reply:
Nice - always great when you can simplify the code.